assets/scss/additional-theme.scss

```diff
@@ -0,0 +1,217 @@
+/**
+SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+**/
+
+.mt-100 {
+  margin-top: 100px;
+}
+
+a:focus,
+.btn:focus,
+.btn:active:focus {
+  box-shadow: none;
+  outline: 0 none;
+}
+
+.section {
+  padding: 100px 0;
+}
+
+.mb-10 {
+  margin-bottom: 10px;
+}
+
+.mb-40 {
+  margin-bottom: 40px;
+}
+
+.heading {
+  text-align: center;
+  color: #777777;
+  padding-bottom: 70px;
+
+  h2 {
+    color: #3c3c3c;
+    padding-bottom: 15px;
+    text-transform: capitalize;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 40px;
+  }
+
+  p {
+    font-size: 17px;
+    line-height: 24px;
+    font-weight: 300;
+  }
+}
+
+.subheading {
+  padding-bottom: 15px;
+  text-transform: capitalize;
```

Comment:
Why do we capitalize each word in sub-headings?
From just my personal opinion I wouldn't do any css-capitalization.

Reply:
We also did that before. Why is the a problem now?

Reply:
We did that before because our template theme did that. I also wasn't a fan of that before, here is just the opportunity to get rid of that.
And I'm not for hard-code capitalization of each word but for normal capitalization in general.

Reply:
Tbh i'm not sure. I'm okay with capitalization of head lines.

Reply:
![image](https://user-images.githubusercontent.com/6124140/103456970-72b68a80-4cfb-11eb-9ef7-370fee97163a.png)
![image](https://user-images.githubusercontent.com/6124140/103456977-7c3ff280-4cfb-11eb-8582-a84f57adc1b4.png)

@DerMolly  what do you think?

Reply:
I like the 2nd one better

Reply:
ok